### PR TITLE
OpenNebula examples

### PR DESCRIPTION
**Pull request by alvarosimon**
Configuration examples to set:
 * ONE server `one.example.org`
 * KVM hyp `hyp01.example.org`
 * AII VM example `vm.example.org`

### CODE REVIEW

**Comment by jrha** [comment]
Can you change all the `.tpl` files to `.pan` please, we deprecated `.tpl` a long time ago.

**Comment by alvarosimon** [comment]
@jrha ah I didn't know, done!

**Comment by jrha** [comment]
Consider replacing all `nlist` with `dict` and removing the `{ }` around `include` statements, both are deprecated.

**Comment by alvarosimon** [comment]
@jrha ok remarks addressed

File: non-grid/basic/common/opennebula/one_auth.pan [hunk 1]
```diff
@@ -0,0 +1,8 @@
+unique template common/opennebula/one_auth;
+
+'/software/components/filecopy/services' =
+  npush(escape("/var/lib/one/.one/one_auth.new"),
```
Comment (jrha) [inline non-grid/basic/common/opennebula/one_auth.pan]: How about this?
```pan
'/software/components/filecopy/services/{/var/lib/one/.one/one_auth.new}' = dict(
```

File: sites/example/site/one/hypervisor.pan [hunk 2]
```diff
@@ -0,0 +1,9 @@
+unique template site/one/hypervisor;
+
+# very very basic
+final variable FILESYSTEM_LAYOUT_CONFIG_SITE = 'site/filesystems/burning';
```
Comment (jrha) [inline sites/example/site/one/hypervisor.pan]: Where does `site/filesystems/burning` come from? What does it look like?

File: non-grid/basic/common/opennebula/one_auth.pan (hunk 1 above)
Comment (alvarosimon) [inline non-grid/basic/common/opennebula/one_auth.pan]: looks much better indeed, I will change it

File: sites/example/site/one/hypervisor.pan (hunk 2 above)
Comment (alvarosimon) [inline sites/example/site/one/hypervisor.pan]: ah copy/paste issue, nothing to do with opennebula component, I will remove this line

**Pull request merged**